tools/pkglint/lib/rules.ts

```diff
@@ -695,6 +695,21 @@ export class JSIIPythonTarget extends ValidationRule {
   }
 }
 
+
+export class JSIIGolangTarget extends ValidationRule {
+  public readonly name = 'jsii/go';
+
+  public validate(pkg: PackageJson): void {
+    if (!isJSII(pkg)) { return; }
+
+    const moduleName = cdkModuleName(pkg.json.name);
+
+    // See: https://github.com/aws/jsii/blob/master/docs/configuration.md#configuring-python
```

Comment:
Bad link - also, the existing documentation does not mention any configuration... Whoop whoop.

```suggestion
    // See: https://aws.github.io/jsii/user-guides/lib-author/configuration/targets/go/
```